dev/io.openliberty.wsoc.internal_fat/fat/src/io/openliberty/wsoc/FATSuite.java

```diff
@@ -28,12 +35,19 @@
  * The classes specified in the @SuiteClasses annotation
  * below should represent all of the test cases for this FAT.
  */
-@SuiteClasses({ BasicTest.class, 
+@SuiteClasses({ BasicTest.class,
                 WebSocket11Test.class
 })
 public class FATSuite {
     private static final Class<?> c = FATSuite.class;
 
+    static Set<String> removeFeatures = new HashSet<>(Arrays.asList("websocket-1.0", "websocket-1.1"));
+    static Set<String> addFeatures = new HashSet<>(Arrays.asList("websocket-2.0"));
+
+    @ClassRule
+    public static RepeatTests repeat = RepeatTests.withoutModification()
```

Comment:
How long does it take to run with the repeat.  Should the withoutModification go with full fat and just the jakartaee9 one be used for lite mode?

Reply:
Updated

Reply:
and reverted because tests have the full fat annotation